### PR TITLE
Mg folijet fix

### PR DESCRIPTION
**Pull request by TetianaParanich**

### CODE REVIEW

File: cypress/integration/data-import/match-on-vendor-reference-number.spec.js [hunk 1]
```diff
@@ -1,4 +1,5 @@
 /* eslint-disable cypress/no-unnecessary-waiting */
+import uuid from 'uuid';
 import getRandomPostfix from '../../support/utils/stringTools';
 import TestTypes from '../../support/dictionary/testTypes';
 import Helper from '../../support/fragments/finance/financeHelper';
```
Comment (penfriend) [inline cypress/integration/data-import/match-on-vendor-reference-number.spec.js:5]: Please fix linter issue:
'FileManager' is defined but never used. Allowed unused vars must match /React/u

File: cypress/integration/data-import/match-on-vendor-reference-number.spec.js [hunk 2]
```diff
@@ -74,35 +99,112 @@ describe('ui-data-import: Match on VRN and update related Instance, Holdings, It
...
+  //   MatchProfiles.deleteMatchProfile(instanceMatchProfileName);
+  //   MatchProfiles.deleteMatchProfile(holdingsMatchProfileName);
+  //   MatchProfiles.deleteMatchProfile(itemMatchProfileName);
+  //   ActionProfiles.deleteActionProfile(instanceActionProfileName);
+  //   ActionProfiles.deleteActionProfile(holdingsActionProfileName);
+  //   ActionProfiles.deleteActionProfile(itemActionProfileName);
+  //   FieldMappingProfiles.deleteFieldMappingProfile(instanceMappingProfileName);
+  //   FieldMappingProfiles.deleteFieldMappingProfile(holdingsMappingProfileName);
+  //   FieldMappingProfiles.deleteFieldMappingProfile(itemMappingProfileName);
+  // });
```
Comment (penfriend) [inline cypress/integration/data-import/match-on-vendor-reference-number.spec.js:165]: Do you need this block commented?

File: cypress/integration/data-import/match-on-vendor-reference-number.spec.js (hunk 1 above)
Comment (penfriend) [reply cypress/integration/data-import/match-on-vendor-reference-number.spec.js:5]: Please fix linter issue:
'Users' is defined but never used. Allowed unused vars must match /React/u         
'InventoryInstance' is defined but never used. Allowed unused vars must match /React/u   
'MatchProfiles' is defined but never used. Allowed unused vars must match /React/u       
'ActionProfiles' is defined but never used. Allowed unused vars must match /React/u      
'FieldMappingProfiles' is defined but never used. Allowed unused vars must match /React/u

File: cypress/integration/requests/requests-sort.spec.js [hunk 3]
```diff
@@ -52,7 +52,7 @@ describe('ui-requests: Sort requests', () => {
     Requests.deleteRequestPolicyApi(requestPolicyId);
   });
```
Comment (penfriend) [inline cypress/integration/requests/requests-sort.spec.js:53]: Please fix linter issue:
'testType' is defined but never used. Allowed unused vars must match /React/u

File: cypress/support/fragments/data_import/match_profiles/newMatchProfile.js [hunk 4]
```diff
@@ -62,10 +62,8 @@ const fillMatchProfileForm = ({
     fillExistingRecordFields(existingRecordFields.in2, 'in2');
     fillExistingRecordFields(existingRecordFields.subfield, 'subfield');
```
Comment (penfriend) [inline cypress/support/fragments/data_import/match_profiles/newMatchProfile.js:63]: please fix linter issue:
'getLongDelay' is defined but never used. Allowed unused vars must match /React/u

**Review by IhorBohdan — APPROVED** [review]

File: cypress/integration/data-import/match-on-vendor-reference-number.spec.js (hunk 1 above)
Comment (TetianaParanich) [reply cypress/integration/data-import/match-on-vendor-reference-number.spec.js:5]: Done

File: cypress/integration/data-import/match-on-vendor-reference-number.spec.js (hunk 2 above)
Comment (TetianaParanich) [reply cypress/integration/data-import/match-on-vendor-reference-number.spec.js:165]: I uncommented this peace of code

File: cypress/integration/requests/requests-sort.spec.js (hunk 3 above)
Comment (TetianaParanich) [reply cypress/integration/requests/requests-sort.spec.js:53]: Done

File: cypress/support/fragments/data_import/match_profiles/newMatchProfile.js (hunk 4 above)
Comment (TetianaParanich) [reply cypress/support/fragments/data_import/match_profiles/newMatchProfile.js:63]: Done

File: cypress/integration/requests/requests-sort.spec.js [hunk 5]
```diff
@@ -52,7 +51,8 @@ describe('ui-requests: Sort requests', () => {
...
 
-  it('C2379 Test Request app sorting (folijet) (prokopovych)', { tags: [testType.smoke, DevTeams.folijet] }, () => {
+  // Test are failed. This is a known issue. That's why test type is deleted.
```
Comment (penfriend) [inline cypress/integration/requests/requests-sort.spec.js:54]: "Test are failed." Maybe you mean IS failed. Just typo :)

**Review by penfriend — APPROVED** [review]

File: cypress/integration/requests/requests-sort.spec.js (hunk 5 above)
Comment (TetianaParanich) [reply cypress/integration/requests/requests-sort.spec.js:54]: yes, it's my bad)

**Pull request merged**